recirq/algorithmic_benchmark_library.py

```diff
@@ -0,0 +1,89 @@
+import dataclasses
+from dataclasses import dataclass
+from functools import lru_cache
+from typing import Type, Callable, List, Tuple
+
+import pandas as pd
+
+from cirq_google.workflow import ExecutableSpec, QuantumExecutableGroup
```

Comment:
From the pytest:

`ModuleNotFoundError: No module named 'cirq_google.workflow'`

Is this waiting on another PR?

Reply:
This and other forthcoming code relies on the currently-unreleased `cirq_google.workflow` module. I'll have to figure out a way to make this work in the mean time. 

1. release openfermion that doesn't pin to a particular cirq version
2. enable testing against multiple versions of cirq including prerelease in ReCirq CI
3. put import guards and conditional testing so these tests only run if a recent enough cirq version is available

Reply:
Following #237 , this is now implemented. The tests only get run on the `next` config until a release of cirq is cut.